core/src/main/java/kafka/server/TierStateMachine.java

```diff
@@ -95,7 +95,7 @@ public TierStateMachine(LeaderEndPoint leader,
     PartitionFetchState start(TopicPartition topicPartition,
                               PartitionFetchState currentFetchState,
                               PartitionData fetchPartitionData) throws Exception {
-        OffsetAndEpoch epochAndLeaderLocalStartOffset = leader.fetchEarliestLocalOffset(topicPartition, currentFetchState.currentLeaderEpoch());
+        OffsetAndEpoch epochAndLeaderLocalStartOffset = leader.fetchEarliestLocalOffset(topicPartition, currentFetchState.getCurrentLeaderEpoch());
```

Comment:
Could you please consider using `currentLeaderEpoch`? it is more readable and it can reduce the changes?